<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CWgtUserList</class>
 <widget class="QWidget" name="CWgtUserList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Users</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableView" name="tvwData"/>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>151</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>151</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="lblPicture">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>130</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="vspPicture">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>242</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="ckbListAll">
        <property name="text">
         <string>&amp;Listar todos</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbAdd">
        <property name="text">
         <string>Adici&amp;onar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
